test/spec/openapi/schema.js

```diff
@@ -110,6 +110,61 @@ test('support 2xx response', async t => {
   t.same(definedPath.responses['3XX'].description, 'Default Response')
 })
 
+test('support multiple content types responses', async t => {
+  const fastify = Fastify()
+  await fastify.register(fastifySwagger, {
+    openapi: true,
+    routePrefix: '/docs',
+    exposeRoute: true
+  })
+
+  const opt = {
+    schema: {
+      response: {
+        200: {
```

Comment:
Could you add a `4xx` response schema with a single content type?

I want to test that there are no conflicts by using the two different syntaxes togheter